libs/daohaus-connect-feature/src/components/ConnectButton/UserConnetedDropdown.tsx

```diff
@@ -1,27 +1,37 @@
 import { useState } from 'react';
+import classNames from 'classnames';
 import { BiChevronDown, BiChevronUp } from 'react-icons/bi';
 import styled, { useTheme } from 'styled-components';
+
 import { getNetworkName, truncateAddress } from '@daohaus/common-utilities';
 import { Button, Dropdown, ParMd, ParXs, ProfileAvatar } from '@daohaus/ui';
 
 import { useHausConnect } from '../../HausConnectContext';
 import { ExplorerLink } from '../ExplorerLink';
-export const UserConnectedDropdown = () => {
+
+export const UserConnectedDropdown = ({ isSm }: { isSm: boolean }) => {
```

Comment:
Slight typo in the file name -- it's currently `UserConetedDropdown` -- missing the 'c'